rust/cloud-storage/worker_trigger/src/service/ecs/mod.rs

```diff
@@ -26,6 +27,10 @@ impl ECSClient {
         cluster: &str,
         subnets: Vec<String>,
     ) -> Result<(), anyhow::Error> {
+        if cfg!(feature = "local") {
```

Comment:
you could just remove the local feature from worker_trigger all together. keep things simpler